### DISCUSSION TITLE
Android 14 solution - use AAAD installer

### DISCUSSION THREAD

**Discussion by MAVERICK23x** [agronick/aa-torque#115] [General]
Can you please have the plug in added to AAAD, It is the only instal method that will work for all users.  

I have tried every other possible method from the discussions but none work with Android 14.  This includes  being in development mode, previous plug in versions and using King installer.  I have a wireless AA dongle but like most it does not have software to enable dev.

 I have just installed performance monitor with AAAD, it is so simple and works flawlessly.

**Comment by agronick** [comment]
It uses the same logic as AAAD for doing the in place update. Someone mentioned they won't add it. I think they're mad that I looked at their source code and used the same method in AA Torque. I would guess that the reason Performance Monitor works is because they targeted an old version of the SDK. I've rewritten all the code the past few months using things that target the new SDKs. Its showing me 51 issues when I try to downgrade so I'm not going to be able to target the old SDKs. Someone is welcome to try and see if it works.

**Comment by MAVERICK23x** [comment]
Hi Kyle,

I am a technical person with a coding background so I understand the
challenges with this not working on the current version of Android which
unfortunately has been forced on many phones over the last several  months.

Upgrade in place is no longer working for any apps but AAAD is using a
different method that I can confirm works as I used it to install
performance monitor and car stream a few days ago.

 I understand it uses old sdks but the good news is I believe it will work
with AA torque as well because the only issue I have is your plugin no
longer show on AA in my vehicle.  This happend with car stream as well but
a reinstall using AAAD has made it available again.


 I have contacted the AAAD developer as well and have a reply from him that
he would infact be interested in add AA torque.

Would you like me reply to him with your contact to get you guys connected
on this ?


I would love to help make this happen to have AA torque available to
everyone.

Thanks

AJ

**Reply by agronick** [reply]
You can send them my email agronick(at)gmail.com. The Github API makes it very easy to get the latest version and they already do that for a bunch of the other apps. They shouldn't need any specific information from me to get it working.

**Reply by hapklaar** [reply]
Curious, did anything come of this? AAAD seems stale, hasn't been updated since 2022 and mentions Android 14 is not supported

**Reply by fcaronte** [reply]
Actually no news, we have only a promise of a new trick from the AAstore Dev but as today no concrete news only a dev subscription but without any reply so actually android box, AAwireless or root are the option

**Reply by hapklaar** [reply]
What do you mean with Android box? I have a Mr12Volt Android box built into my BMW E91 CIC.

PS seems aastore has a new version in Beta
![image](https://github.com/agronick/aa-torque/assets/7703320/8733fee8-9036-40d1-adfe-4202aa79bc71)

**Reply by fcaronte** [reply]
Yes Is what I've say is a big "seem" actually no one seem to get this beta, then for the box yes I mean like the carlinkit box or other thing using Carplay protocol to run a full android system working without the smartphone

**Comment by PhilDoGGo** [comment]
The only way I've got this to work, is using an AAwireless device in developer mode.

I had issues, after upgrading the Android 14 and using the wireless Motorola MA1 - I tried everything, including different installers etc, so I bought an AAwireless and put it in developer mode and it now works great, although it doesn't appear as an app in the Android Auto settings on my Samsung phone, but appears fine on my car's head unit.
I tried just using a standard cable connection, but that didn't work either, so this is the only combination I know of.

**Comment by fcaronte** [comment]
Today I check if really aaad can make work performance monitor but is not true, it show only the icon as for some other app but won't showing in the car... As I remember we use the same method on kinginstaller so can't give different results.

In this day I try aagateway and was working to unlock A14 app exactly as for AAwireless, unfortunately you need to get XDA Version because the one you can find on GitHub is very old version and it doesn't have the unlock and dpi trick, is only a wireless connection.
Then unfortunately I come back to carisifi because aagateway connection is very unstable and most of the time won't start itself, I was using an S7 as car phone and connect wireless to my S24ultra, yes was working but really unstable, unfortunately...

I read aastore Dev say he found an A14 trick and will update his aastore app. I hope will not be one year long as when it say was found the Oppo trick 😬![IMG_20240224_124434.jpg](https://github.com/agronick/aa-torque/assets/26366403/3d20548b-c0b9-4e16-a0ea-8ece8e8db004)
